services/wallet/api.go

```diff
@@ -376,26 +397,31 @@ func (api *API) GetEthereumChains(ctx context.Context) ([]*network.CombinedNetwo
 	return api.s.rpcClient.NetworkManager.GetCombinedNetworks()
 }
 
+// @deprecated
 func (api *API) FetchPrices(ctx context.Context, symbols []string, currencies []string) (map[string]map[string]float64, error) {
```

Comment:
I see this is used by Nim's `token.Service.getTokenPrice`

Reply:
We should have everything we need inside the wallet token?

Reply:
I believe this was previously used to fetch the current price in the send modal if the cached price is older than X. Not sure if that's still the case though